crates/tx5-connection/src/webrtc/go_pion.rs

```diff
@@ -124,8 +148,25 @@ async fn task(
             use tx5_go_pion::PeerConnectionEvent as Evt;
             while let Some(evt) = peer_evt.recv().await {
                 match evt {
-                    Evt::Error(_) => break,
-                    Evt::State(_) => (),
+                    Evt::Error(err) => {
+                        cmd_send2.send_or_close(Cmd::Error(
+                            std::io::Error::other(format!(
+                                "Pion error event: {err}"
+                            )),
+                        ))?;
+                    }
+                    Evt::State(state) => match state {
+                        tx5_go_pion::PeerConnectionState::Disconnected => {
+                            cmd_send2.send_or_close(Cmd::Close)?;
+                        }
+                        tx5_go_pion::PeerConnectionState::Closed => {
+                            cmd_send2.send_or_close(Cmd::Close)?;
+                        }
+                        tx5_go_pion::PeerConnectionState::Failed => {
+                            cmd_send2.send_or_close(Cmd::Close)?;
+                        }
```

Comment:
I was just after changing the `Disconnected` and `Closed` events, is this one not an error? It's not very clear from their docs but I'd expect it to be

Reply:
Yeah I agree its not very clear from the spec. Let's go with error https://github.com/holochain/tx5/pull/194/commits/e84fceae60304513ecf80c40a7ad3a99cba61878